R/parse.R

```diff
@@ -149,6 +149,12 @@ parse_rprof <- function(path = "Rprof.out", expr_source = NULL) {
       list(filename = filename, content = content)
     }, SIMPLIFY = FALSE, USE.NAMES = FALSE)
 
+  # Assign absolute paths to provide the original location when raising source events
+  file_contents <- lapply(file_contents, function(e) {
+    suppressWarnings(e$normpath <- normalizePath(e$filename))
```

Comment:
Actually, I think the `normalizePath` won't work if `filename` is a path that's not in the homedir (like in a package). I think if you test this with a package installed with source refs, that's what you'll find.

The original full path is available above; it gets trimmed in line 122, by `trim_filenames()`, so you could preserve it up there.